tests/support/cluster_util.tcl

```diff
@@ -112,7 +148,7 @@ proc start_cluster {masters replicas options code {slot_allocator continuous_slo
     # Configure the starting of multiple servers. Set cluster node timeout
     # aggressively since many tests depend on ping/pong messages. 
     set cluster_options [list overrides [list cluster-enabled yes cluster-ping-interval 100 cluster-node-timeout 3000]]
-    set options [concat $cluster_options $options]
+    set options [merge_options $options $cluster_options]
```

Comment:
Are we overcomplicating this, should we instead just do something like:

```suggestion
    foreach {key value} $options {
        if {$key eq "overrides"} {
            set options lreplace $options [lsearch $options -exact $value] [concat [list cluster-enabled yes cluster-ping-interval 100 cluster-node-timeout 3000] $overrides]
        }
    }
```

EDIT: I'm testing if this is actually the right code.

Reply:
I think I'm lost about what this change is trying to do then.